sources/install/package_web.sh

```diff
@@ -357,8 +357,10 @@ function install_eyewitness() {
 
 function install_oneforall() {
     colorecho "Installing OneForAll"
-    git -C /opt/tools/ clone --depth 1 https://github.com/shmilylty/OneForAll.git
+    git -C /opt/tools/ clone --depth 1 https://github.com/shmilylty/OneForAll.git 
     cd /opt/tools/OneForAll
+    cp -v /root/sources/assets/patches/OneForAll.patch OneForAll.patch
+    git apply --verbose OneForAll.patch
```

Comment:
Can you comment on why this changes is needed?

Reply:
no longer necessary